opentreemap/treemap/views/map_feature.py

```diff
@@ -101,13 +102,48 @@ def _map_feature_detail_context(request, instance, feature_id, edit=False):
 
     if feature.is_plot:
         partial = 'treemap/partials/plot_detail.html'
+        _add_plot_field_groups(context, instance)
     else:
         app = feature.__module__.split('.')[0]
         partial = '%s/%s_detail.html' % (app, feature.feature_type)
 
     return context, partial
 
 
+def _add_plot_field_groups(context, instance):
+    templates = {
+        "tree.id": "treemap/field/tree_id_tr.html",
+        "tree.species": "treemap/field/species_tr.html",
+        "tree.diameter": "treemap/field/diameter_tr.html"
+    }
+
+    labels = {
+        "tree.species": "plot-species",
+        "tree.diameter": _("Trunk Diameter")
+    }
+    labels.update({
+        v: k for k, v in context['tree'].scalar_udf_names_and_fields})
+    labels.update({
+        v: k for k, v in context['plot'].scalar_udf_names_and_fields})
+
+    def info(group):
+        group['fields'] = [
+            (field, labels.get(field),
+             templates.get(field, "treemap/field/tr.html"))
+            for field in group.get('field_keys', [])
+        ]
+        group['collection_udfs'] = [
+            next(udf for udf in udf_defs(instance)
```

Comment:
I considered writing defensive code here to protect from that, but decided it wasn't worth it.  If we start making `Instance`s that lack Stewardship, a _lot_ of things will blow up all over the code base